alerter-component/src/main/java/io/github/steliospaps/ighackathon/instrumentpricealerter/alertercomponent/InitialTableScanner.java

```diff
@@ -49,7 +42,7 @@
 
 	@EventListener(ApplicationReadyEvent.class)
 	void scanForTriggers() {
-		log.info("scanning entities scanChunk={} ***********************", scanChunk);
+		log.info("scanning entities for triggers scanChunk={} ***********************", scanChunk);
```

Comment:
for now we are scanning triggers, but in reality we scan everything in the table. So I think the 'for triggers' here might be misleading. Actually the name of the function should be changed.

Reply:
will update method name and log line